appserver/admingui/SNMP-notifier-console-plugin/src/main/resources/SNMP/SNMPNotifierTabs.jsf

```diff
@@ -0,0 +1,28 @@
+<!--
+
+ DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
+
+ Copyright (c) 2017 Payara Foundation and/or its affiliates. All rights reserved.
+
+ The contents of this file are subject to the terms of the Common Development
+ and Distribution License("CDDL") (collectively, the "License").  You
+ may not use this file except in compliance with the License.  You can
+ obtain a copy of the License at
+ https://glassfish.dev.java.net/public/CDDL+GPL_1_1.html
+ or packager/legal/LICENSE.txt.  See the License for the specific
+ language governing permissions and limitations under the License.
+
+ When distributing the software, include this License Header Notice in each
+ file and include the License file at packager/legal/LICENSE.txt.
+ -->
+<sun:tab id="notificationSNMPTab" immediate="true" text="$resource{i18nsnn.notifier.SNMP.tabs.tabText}" 
+         toolTip="$resource{i18nsnn.notifier.SNMP.tabs.tabToolTip}">
+    <!beforeCreate
+        setResourceBundle(key="i18nsnn" bundle="fish.payara.admingui.notifier.SNMP.Strings");
+        setResourceBundle(key="i18nc" bundle="org.glassfish.common.admingui.Strings");
+    />
+    <!command
+        setSessionAttribute(key="notificationConfigTab" value="notificationSNMPTab");
+        gf.redirect(page="#{request.contextPath}/SNMPNotifier/SNMP/SNMPNotifierConfiguration.jsf?configName=#{pageSession.configName}");
```

Comment:
Note if you're changing the ID in console-config, you'll need to change SNMPNotifier to match the new ID.